kola/tests/ignition/luks.go

```diff
@@ -0,0 +1,79 @@
+// Copyright The Mantle Authors
+// Copyright 2020 Red Hat
+// SPDX-License-Identifier: Apache-2.0
+package ignition
+
+import (
+	"github.com/coreos/go-semver/semver"
+	"github.com/flatcar-linux/mantle/kola/cluster"
+	"github.com/flatcar-linux/mantle/kola/register"
+	"github.com/flatcar-linux/mantle/platform"
+	"github.com/flatcar-linux/mantle/platform/conf"
+	"github.com/flatcar-linux/mantle/platform/machine/qemu"
+	"github.com/flatcar-linux/mantle/platform/machine/unprivqemu"
+)
+
+func init() {
+
+	cfg := conf.Ignition(`{
+		  "ignition": {"version": "3.2.0"},
+		  "storage": {
+		    "luks": [{
+		      "name": "data",
+		      "device": "/dev/vdb"
```

Comment:
Just an idea if we want to avoid tying it to qemu, we could also use `/dev/disk/by-partlabel/USR-B` or `/dev/disk/by-partlabel/OEM-CONFIG`.
Later when clevis and tpm2 may be added we either have to add a software tpm to qemu or use another cloud platform.

Reply:
`/dev/disk/by-partlabel/USR-B` sounds good - let's try that. Thanks for the suggestion !